### DISCUSSION TITLE
Row groups

### DISCUSSION THREAD

**Discussion by david-waterworth** [posit-dev/great-tables#332] [Q&A] [answered]
I have data generated using polars `group_by`, i.e. df.group_by("c1","c2",c3").agg(pl.col("c4").sum(), pl.col("c5").sum())'

I'd like to format so that the groups are easier to distinguish - the simplest way is in columns "c1" .. "c3" only render a value if it's the same as the value above. Is this possible? Alternatively some sort of grouping like the exibble example (https://posit-dev.github.io/great-tables/blog/introduction-0.3.0/ ) but using multi-column row lables

Is something like this supported?

**Comment by jrycw** [comment, marked as answer]
@david-waterworth,  is #345 the feature you're looking for?

**Comment by jrycw** [comment]
I'll close the discussion since the issue appears to be resolved.
